libs/wingc/src/jsify.rs

```diff
@@ -1371,6 +1372,11 @@ impl<'a> JSifier<'a> {
 			code.append(" {");
 			code.indent();
 
+			// TODO Hack to make sure closures match the IInflight contract from wingsdk
+			if class_type.is_closure() {
```

Comment:
is this still needed?

Reply:
See https://github.com/winglang/wing/pull/4993#discussion_r1402488116